projects/jupyter-server-ydoc/jupyter_server_ydoc/handlers.py

```diff
@@ -291,6 +293,17 @@ async def on_message(self, message):
         """
         On message receive.
         """
+        message_type = message[0]
+        if message_type == MessageType.UPDATE:
+            msg = message[2:].decode("utf-8")
```

Comment:
Could you remind me why it's not:
```suggestion
            msg = message[1:].decode("utf-8")
```

Reply:
Because `message[1]` contains the length of the string (written by `writeVarString`), so you must skip it to start decoding the actual string.

Reply:
I think that we should use a [Decoder](https://github.com/y-crdt/pycrdt/blob/7f9de2a87c38668b97585d778258912cbf9e4e6b/python/pycrdt/_sync.py#L163) to decode that message.